GamepadEventDrivenInputAPI/explainer.md

```diff
@@ -86,24 +86,36 @@ window.addEventListener('gamepadconnected', () => {
 
 ## Goals
 
-Reduce input latency by moving away from constant polling and introducing event-driven input handling. 
+Reduce input latency by moving away from constant polling and introducing event-driven input handling.
 
 ## Non-goals
```

Comment:
nit: be consistent with the title format. Choose one style and stick with it: either all words starting with uppercase or just the first word?